client/landing/jetpack-cloud/components/stats-footer/style.scss

```diff
@@ -0,0 +1,49 @@
+.stats-footer {
+	border-top: 1px solid var( --color-neutral-5 );
+	color: var( --color-neutral-80 );
+	display: flex;
+	padding-top: 10px;
+	margin-top: 40px;
+	flex-wrap: wrap;
+}
+
+.stats-footer__header {
+	color: var( --color-neutral-50 );
+	font-size: 13px;
```

Comment:
Where did you get that `13px`? Does this come from the Figma file? I'm curious about how close we should follow the designs.

Reply:
I got it from the Figma, yeah! You can select elements to see CSS details.

As for how closely, as a rule of thumb I round spacing, but keep to font sizes. YMMV?